### PR TITLE
Dockerfiles cleanup & Travis CI support

### PR DESCRIPTION
**Pull request by fidencio**
@jensfr, @harche,

This PR consists in two different patch sets, where the first set is focused on the Dockerfile improvement, while the second adds support for using Travis CI.

**Dockerfile**
* Remove kata-shim after downloading it
  + As we're using shimv2, there's no need to install the kata-shim package;
* Rename advanced virt .repo file
  + Let's give the .repo file a more descriptive name, as we'll be using more .repo files coming from Virt SIG;
* Use kata packages from Virt SIG
  + Let's drop the dependency on my COPR repo and start relying on CentOS, which provides us support for PPC64LE for free;
* Don't copy SIG Virt repos to the final container
  + The final container should have only the needed bits we downloaded, avoiding errors due to non planned updates in the packages shipped;
* Add ARCH as a build argument
  + This will allow us to easily build for PPC64LE (and other architectures of our interest)

**Travis CI**
* Build & update the image using Travis CI
  + This commit requires a some configurations done beforehand such as:
    - Travis CI must be enabled by the owner of the repo;
    - QUAY_USERNAME and QUAY_PASSWORD must be set, as secrets, on Travis CI   side;
  + And also an agreement about the branches naming that will be adopted and will be used as basis for the image name, such as:
    - ocp-x.y for the images that should be used together with OCP x.y,  generating images named "x.y-$arch";
    - master will generate images names "latest-$arch";
    - any other branch name will not trigger Travis CI;

**Notes**
* A similar PR has to be provided to the other repos, you can easily take this one as basis for the work that has to be done. 
  + Please, let me know whether you guys can take care of updating the needed bits there or if I should submit such PR;
* The code, everywhere, must be updated so it becomes robust enough to deal with the proposed naming for the images.
  + This is what makes me think that could be easier for you guys to adapt the other repos and give them a try;

### CODE REVIEW

**Comment by bpradipt** [comment]
> @jensfr, @harche,
> 
> This PR consists in two different patch sets, where the first set is focused on the Dockerfile improvement, while the second adds support for using Travis CI.
> 
> **Dockerfile**
> 
> * Remove kata-shim after downloading it
>   
>   * As we're using shimv2, there's no need to install the kata-shim package;
> * Rename advanced virt .repo file
>   
>   * Let's give the .repo file a more descriptive name, as we'll be using more .repo files coming from Virt SIG;
> * Use kata packages from Virt SIG
>   
>   * Let's drop the dependency on my COPR repo and start relying on CentOS, which provides us support for PPC64LE for free;
> * Don't copy SIG Virt repos to the final container
>   
>   * The final container should have only the needed bits we downloaded, avoiding errors due to non planned updates in the packages shipped;
> * Add ARCH as a build argument
>   
>   * This will allow us to easily build for PPC64LE (and other architectures of our interest)
> 
Awesome.. thanks so much @fidencio

**Review by jensfr — APPROVED** [review]
This looks good to me. I'll have to adapt kata-operator-daemon to make use of the new form of multi-arch tags

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,8 @@
-FROM centos:8.2.2004 as katapackages
+ARG ARCH=amd64
+FROM ${ARCH}/centos:8.2.2004 as katapackages
```
Comment (bpradipt) [inline Dockerfile]: Hi @fidencio the ARCH argument is not mandatory and can be removed. Centos image in dockerhub is multi-arch. So using the image without any arch prefix will work for both Intel and Power. 
For example on ppc64le
```
docker run -it centos:8.2.2004 uname -m
ppc64le
```
on amd64 
```
docker run -it centos:8.2.2004 uname -m
x86_64
```
A good description is available in the following [link](https://developer.ibm.com/tutorials/createmulti-architecture-docker-images/)

I have also tested building the container image on standalone ppc64le successfully using your changes.

Comment (fidencio) [reply Dockerfile]: Cool, I've dropped the ARCH and re-worked this series to use buildx.

Thanks for pointing this out!

**Comment by fidencio** [comment]
I've reworked the series using buildx. @bpradipt, @harche, @jensfr, please, take a look.

**Review by jensfr — APPROVED** [review]

**Pull request merged**